dashboard/src/features/transactions/components/Summary/Summary.scss

```diff
@@ -27,13 +27,15 @@
   background: none;
 }
 
-.odd td {
-  background: $background-emphasis-color;
+td {
+  border-bottom: 1px solid #f2f2f2;
```

Comment:
We just removed a color of #f2f2f2 on line https://github.com/chain/chain/pull/162/files#diff-027ae693252442f625c31b3c345afdd3L13 - should this be `$background-content-color` instead?